aws/aws.go

```diff
@@ -478,6 +487,44 @@ func GetAllResources(targetRegions []string, excludeAfter time.Time, resourceTyp
 		count++
 	}
 
+	// Global Resources - These resources are global and do not belong to a specific region
+
+	logging.Logger.Infof("Checking region [%d/%d]: %s", count, totalRegions, GlobalRegion)
+
+	// As there is no actual region named global we have to pick a random one just to create the session
+	randomRegion, err := getRandomRegion()
+	if err != nil {
+		return nil, errors.WithStackTrace(err)
+	}
+
+	session, err := session.NewSession(&awsgo.Config{
+		Region: awsgo.String(randomRegion)},
+	)
+	if err != nil {
+		return nil, errors.WithStackTrace(err)
+	}
+
+	globalResources := AwsRegionResource{}
+
+	// IAM Users
+	iamUsers := IAMUsers{}
+	if IsNukeable(iamUsers.ResourceName(), resourceTypes) {
```

Comment:
NIT: I have had a look at the whole function, and it is getting very long. It's not for this PR, but was wondering if we could somehow refactor the overall function `GetAllResources`?

Reply:
Absolutely! 337 lines for a single function is too much :smile: 
And, as you said, maybe not on this PR.